src/aquifer.default.json

```diff
@@ -6,6 +6,7 @@
     "settings": "settings",
     "build": "build",
     "root": "root",
+    "drush": "drush",
```

Comment:
I feel it should be, it creates an empty placeholder. Very common need and I'd consider it a best practice.